### ISSUE TITLE
The question of whether the loss function can be used in self supervised depth estimation

### ISSUE THREAD

**Issue by chaoying0115** [lorenzopapa5/METER#1]
Thank you for your team's excellent work. May I ask if the BLF loss function proposed in the paper can be applied to the problem of self supervised depth estimation? Thank you!

**Comment by lorenzopapa5** [comment]
It depends on how the ground truth data distribution used during the training process is represented; however, generally speaking, it is possible to use the BLF function for the given task.
